src/cobalt_strike.yml

```
@@ -21,6 +21,20 @@
           - python3-boto3
     - name: Install some Cobalt Strike addons
       block:
+        # This tool requires a JRE, but we have already installed one
+        # above
+        - name: Install C2concealer
+          ansible.builtin.import_role:
+            name: assessment_tool
+          vars:
+            archive_src: "https://github.com/FortyNorthSecurity/C2concealer/\
+            tarball/master"
+            install_dir: /tools/C2concealer
+            pip_extra_args: "-e"
```

I would rather use the long form so it's more obvious what's going on here.

```suggestion
            pip_extra_args: "--editable"
```